freepbx/initdb.d/migration.php

```diff
@@ -75,6 +67,30 @@
 	$stmt->execute(array_column($res, 'extension'));
 }
 
+/* Set outbound_proxy to all physical and mobile extensions to be used with proxy */
+$sql = "SELECT extension,
+        IF (`asterisk`.`sip`.`data` = 'sdes', true, false) AS `srtp`
+        FROM `asterisk`.`rest_devices_phones`
+        JOIN `asterisk`.`sip`
+        ON `asterisk`.`rest_devices_phones`.`extension` = `asterisk`.`sip`.`id`
+        WHERE ( `type` = 'physical' OR `type` = 'mobile' OR `type` = 'customphysical' )";
+
+$stmt = $db->prepare($sql);
+$stmt->execute();
+$res = $stmt->fetchAll(\PDO::FETCH_ASSOC);
+
+if (count($res) > 0) {
+	$qm_string = str_repeat('?, ', count($res) - 1) . '?';
+
+	# configure proxy on all FreePBX extensions
+	$proxy_value = 'sip:'.$_ENV['PROXY_IP'].':'.$_ENV['PROXY_PORT'].';lr';
+
+	$sql = "UPDATE `asterisk`.`sip` SET `data` = ? WHERE `keyword` = 'outbound_proxy' AND `id` IN ($qm_string)";
+	$stmt = $db->prepare($sql);
+	$stmt->execute(array_merge([$proxy_value], array_column($res, 'extension')));
+}
+
+
```

Comment:
Also do that for pjsip trunks
```suggestion
$sql = "UPDATE `asterisk`.`pjsip`
	SET `pjsip`.`data` = ?
	WHERE `pjsip`.`keyword` = 'outbound_proxy'";

$stmt = $db->prepare($sql);
$stmt->execute(['sip:'.$_ENV['PROXY_IP'].':'.$_ENV['PROXY_PORT'].';lr'];
```